src/com/palmergames/bukkit/towny/object/Nation.java

```diff
@@ -565,21 +566,6 @@ public boolean hasResident(String name) {
 		return false;
 	}
 
-    @Override
-    protected World getBukkitWorld() {
-        if (hasCapital() && getCapital().hasWorld()) {
-            return BukkitTools.getWorld(getCapital().getWorld().getName());
-        } else {
-            return super.getBukkitWorld();
-        }
-    }
-
-
-	@Override
-	public String getEconomyName() {
-		return StringMgmt.trimMaxLength(Nation.ECONOMY_ACCOUNT_PREFIX + getName(), 32);
-	}
-
```

Comment:
Why are these getting nuked? There will certainly be plugins using the getEconomyName() to manipulate the eco account.